Makefile

```diff
@@ -57,7 +55,7 @@ install: install-binary install-docs
 	# TODO(runcom)
 	#install -m 644 completion/bash/skopeo ${BASHINSTALLDIR}/
 
-install-binary: skopeo
+install-binary: ./skopeo
```

Comment:
It does nothing wrt typical read/write/does the file exist type operations. The `./` only holds meaning on execution, to resolve which path which executable we want to execute type issues.  It's just a habitual thing in this context :-)